react/react2/week1/react2week1/src/GithubSearch.js

```diff
@@ -0,0 +1,20 @@
+import React, { useState, createContext } from "react";
+import { FetchData } from "./FetchData";
+export const SearchContext = createContext();
+export function GithubSearch() {
+  const [searchInput, setSearchInput] = useState("");
+  console.log(searchInput);
+  //   const user = useContext();
+  return (
+    <div>
+      <input
+        type="text"
+        value={searchInput}
+        onChange={(e) => setSearchInput(e.target.value)}
+      ></input>
+      <SearchContext.Provider value={searchInput}>
```

Comment:
It doesn't really make sense to  return the SearchContext in this component. It is an input component and can't really be parent of anything. Instead you should use the callback pattern to send the input up into the context and then have the list react to the change in the context. 
So, the context is a parent of the list component and the input component.